examples/pxScene2d/src/pxScene2d.cpp

```diff
@@ -1681,7 +1681,12 @@ rtError pxScene2d::logDebugMetrics()
 {
   script.garbageCollect();
   rtLogInfo("pxobjectcount is [%d]",pxObjectCount);
+
+#ifdef PX_PLATFORM_MAC
+  rtLogInfo("texture memory usage is [%lld]",context.currentTextureMemoryUsageInBytes());
```

Comment:
Let's change %lld to PRId64.  This is the desired define to use to print int64_t, and it will eliminate the need for a platform #ifdef here.